lib/default.nix

```diff
@@ -101,16 +101,16 @@ let
       cleanSource sourceByRegex sourceFilesBySuffices
       commitIdFromGitRepo cleanSourceWith pathHasContext
       canCleanSource;
-    inherit (modules) evalModules closeModules unifyModuleSyntax
-      applyIfFunction mergeModules
+    inherit (modules) evalModules collectModules unifyModuleSyntax
+      fullyUnifyModuleSyntax applyIfFunction mergeModules
```

Comment:
I don't think we should reexport these functions.

Reply:
Sounds reasonable. Also they'll be available through `lib.modules.*` anyways